lib/Digitick/Sepa/DomBuilder/CustomerCreditTransferDomBuilder.php

```diff
@@ -81,6 +81,10 @@ public function visitPaymentInformation(PaymentInformation $paymentInformation)
         );
 
         $paymentTypeInformation = $this->createElement('PmtTpInf');
+        if($paymentInformation->getInstructionPriority()) {
```

Comment:
Can you align your coding style here and place a space behind the if

Reply:
Hi @monofone and thanks for your feedback. Both issues fixed now.